eclipse-scout-core/src/tile/TileGrid.ts

```diff
@@ -1165,18 +1207,18 @@ export class TileGrid extends Widget implements TileGridModel {
 
   findTileIndexAt(x: number, y: number, startIndex?: number, reverse?: boolean): number {
     startIndex = scout.nvl(startIndex, 0);
-    return arrays.findIndexFrom(this.filteredTiles, startIndex, (tile, i) => {
+    return arrays.findIndexFrom(this._filteredTiles, startIndex, (tile, i) => {
       return tile.gridData.x === x && tile.gridData.y === y;
     }, reverse);
   }
 
   /**
    * If the max range is used, the live list of filtered tiles is returned, because every tile has to be in the range.
    */
-  findTilesInRange(viewRange: Range, filter?: Predicate<Tile>): Tile[] {
+  findTilesInRange(viewRange: Range, filter?: Predicate<TTile | PlaceholderTile>): (TTile | PlaceholderTile)[] {
```

Comment:
Should the predicate really get placeholders? For other public methods the placeholders are like not existing. But here developers have to deal with them. Maybe never include placeholders in the find and therefore also not add them to the predicate and return API?
Same maybe also applies to method "eachTileInRow" below.